src/main/java/sg/ncl/MainController.java

```diff
@@ -2283,11 +2301,24 @@ private void setSessionVariables(HttpSession session, String loginEmail, String
         session.setAttribute("sessionLoggedEmail", loginEmail);
         session.setAttribute("id", id);
         session.setAttribute("name", user.getFirstName());
+        // FIXME: get user type from token
+//        session.setAttribute("userType", "normal");
+        session.setAttribute("userType", "admin");
     }
 
     private void removeSessionVariables(HttpSession session) {
         session.removeAttribute("sessionLoggedEmail");
         session.removeAttribute("id");
         session.removeAttribute("name");
+        // FIXME: get user type from token
+        session.removeAttribute("userType");
+    }
+
+    private boolean validateIfAdmin(HttpSession session) {
+        if (session.getAttribute("userType").toString().equals("admin")) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "admin" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1132)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this if-then-else statement by a single return statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1126)